build.gradle.kts

```
@@ -25,3 +25,9 @@ plugins {
     alias(libs.plugins.kotlin.serialization) apply false
     alias(libs.plugins.dokka) apply false
 }
+
+buildscript {
+    dependencies {
+        classpath(libs.dokka.versioning)
```

Might be worth to add these explanatory comments:
```suggestion
        // there doesn't appear to be a better way to provide this to subprojects.
        // this is what lets us put the version number dropdown list in the generated dokka.
        // it is a "dokka plugin" which is not a gradle plugin, it needs to be on the classpath
        // before any dependent subproject uses its symbols to configure a dokka task.
        classpath(libs.dokka.versioning)
```